src/main/kotlin/io/provenance/client/grpc/PbClient.kt

```diff
@@ -167,53 +170,4 @@ open class PbClient(
         gasAdjustment = gasAdjustment,
         feeGranter = feeGranter
     ).let { baseReq -> broadcastTx(baseReq, estimateTx(baseReq), mode) }
-
-    fun getAcountBalance(bech32Address: String, denom: String): CoinOuterClass.Coin =
-        bankClient.balance(
-            cosmos.bank.v1beta1.QueryOuterClass.QueryBalanceRequest.newBuilder().setAddress(bech32Address).setDenom(denom).build()
-        ).balance
-
```

Comment:
Was all of this moved into the Provenance protobuf binding code?

Reply:
this was added in that last pr yesterday so hopefully doesnt need to stay

Reply:
Some, but not all was moved ([pr](https://github.com/provenance-io/provenance/pull/664)).

This logic was removed primarily because it was only being used for tests which have also been removed.  The setup to get the tests running wasn't straightforward and reverting back to @vwagner's original tests was sufficient.